pkg/addonmanager/controllers/cmamanagedby/controller.go

```diff
@@ -17,11 +17,15 @@ import (
 )
 
 const (
-	controllerName = "management-addon-controller"
+	controllerName = "cma-managed-by-controller"
 )
 
-// clusterManagementAddonController reconciles cma on the hub.
-type clusterManagementAddonController struct {
+// cmaManagedByController reconciles clustermanagementaddon on the hub
```

Comment:
Is it necessary to remove this? will remove this controller later when addon manager no longer honor this annotation?